updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc

```diff
@@ -114,24 +114,27 @@ and must be no more than 63 characters`, try creating the Update Service with a
 ====
 
 // Configuring the Cluster Version Operator (CVO)
-include::modules/update-service-configure-cvo.adoc[leveloffset=+3]
+include::modules/update-service-configure-cvo.adoc[leveloffset=+1]
 
 [NOTE]
 ====
 See xref:../../../networking/enable-cluster-wide-proxy.adoc#enable-cluster-wide-proxy[Configuring the cluster-wide proxy] to configure the CA to trust the update server.
 ====
 
+// Configuring the Cluster Version Operator (CVO) to retrieve release image signatures
+include::modules/configure-cvo-retrieve-image-signatures.adoc[leveloffset=+2]
+
 [id="next-steps_updating-restricted-network-cluster-osus"]
 == Next steps
 
 Before updating your cluster, confirm that the following conditions are met:
 
 * The Cluster Version Operator (CVO) is configured to use your locally-installed OpenShift Update Service application.
-* The release image signature config map for the new release is applied to your cluster.
```

Comment:
original signature config map is still support, so i think we should not remove it, right? so suggest to add `or the cluster is able to retrieve release image signatures for the update target release images from osus`

Reply:
I can keep the old statement while including the new option in the same sentence, thanks